server/muxer/muxer.go

```diff
@@ -18,6 +18,8 @@ import (
 	"fmt"
 	"net"
 
+	ulog "github.com/tigrisdata/tigris/util/log"
+
```

Comment:
so remove the extra line and run `go imports` applies to other files as well.

Reply:
Ack, I can do that. Do you think there is an issue with my GoLand settings that it adds these imports with an extra newline?